src/components/headers/collection-header.tsx

```diff
@@ -512,6 +563,109 @@ export class CollectionHeader extends React.Component<IProps, IState> {
     });
   };
 
+  private signCollection = () => {
+    const errorAlert = (status: string | number = 500): AlertType => ({
+      variant: 'danger',
+      title: t`API Error: ${status}`,
+      description: t`Failed to sign all versions in the collection.`,
+    });
+
+    this.setState({
+      alerts: [
+        ...this.state.alerts,
+        {
+          id: 'loading-signing',
+          variant: 'info',
```

Comment:
```suggestion
          variant: 'success',
```
So far as I know, we use the green 'success' alert for the successful start to any action, so I would go with that (unless @himdel says otherwise) :)

Reply:
meeting.

Reply:
Confirming, we're going with success

Reply:
@himdel  I had a discussion with @ShaiahWren where we agreed on a meeting for today (which was pushed to next week), hence the meeting comment. I also made clear to @ShaiahWren that I don't mind using success.